tests/ui/lint/large_assignments/copy_into_box_rc_arc.rs

```diff
@@ -0,0 +1,38 @@
+#![deny(large_assignments)]
+#![feature(large_assignments)]
+#![move_size_limit = "1000"]
+// build-fail
+// only-x86_64
```

Comment:
Can we relax this so they run on other 64 bit archs (like aarch64 especially) too. People develop on aarch64, and not being able bless these is annoying.

Reply:
I can't think of why that wouldn't work, so I pushed a commit for it now.